imports/plugins/included/payments-stripe/server/methods/stripe.js

```diff
@@ -97,22 +97,45 @@ Meteor.methods({
     });
     check(paymentData, {
       total: String,
-      currency: String
+      currency: String,
+      shopId: String
     });
 
-    const chargeObj = {
-      amount: "",
-      currency: "",
-      card: {},
-      capture: true
-    };
+    // check if this is a seller shop for destination and transaction fee logic
+    const sellerShop = sellerShops.findOne(paymentData.shopId);
+
+    if (sellerShop && sellerShop.stripeConnectSettings) {
+      const chargeObj = {
```

Comment:
This `const chargeObj` will be scoped to the `if (sellerShop && sellerShop.stripeConnectSettings) {` block and will not be accessible outside that scope, so this code will not work like you expect it to.